plutarch-test/src/Plutarch/MonadicSpec.hs

```diff
@@ -0,0 +1,49 @@
+{-# LANGUAGE OverloadedRecordDot #-}
+{-# LANGUAGE QualifiedDo #-}
+
+module Plutarch.MonadicSpec (spec) where
+
+import Test.Syd
+
+import Plutus.V1.Ledger.Api
+
+import Plutarch.Api.V1
```

Comment:
For the future, please try to avoid implicit imports like this. Since this isn't important code, we shouldn't block the PR on this though.

Reply:
Note that I do know that the tests originally had this problem due to laziness on my part.

Reply:
Sure. I assume we can relax on `Plutus.V1.Ledger.Api` and the like for testing code. Added explicit imports for Plutarch.*.